docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -970,6 +971,24 @@ The values of the `typescript.options.lib` compiler option should be identical t
 *Related configuration file property*: [compilerOptions](configuration-file.md#compileroptions).
 *Related API method*: [runner.compilerOptions](testcafe-api/runner/compileroptions.md)
 
+### --cache
+
+If enabled, TestCafe proxy caches webpage dependencies (like styles and scripts) for webpages visited during tests. When TestCafe revisits the page, it pulls assets from the cache instead of requesting them from the server.
```

Comment:
Uniform phrasing

```suggestion
If enabled, the TestCafe proxy caches webpage assets (such as stylesheets and scripts) for the webpages that it processes. The next time the proxy accesses the page, it pulls assets from its cache instead of requesting them from the server.
```